pandas/core/generic.py

```diff
@@ -3287,7 +3287,8 @@ def _consolidate(self, inplace=False):
 
     def consolidate(self, inplace=False):
         """
-        DEPRECATED: consolidate will be an internal implementation only.
+        .. deprecated:: 0.20.0
+           Consolidate will be an internal implementation only.
```

Comment:
Same comment here as previous. By putting this deprecated directive on the first line, it gets less informative in a api summary table